appscale/tools/node_layout.py

```diff
@@ -602,15 +645,11 @@ def generate_cloud_layout(self):
       Returns:
         A dict that has one controller node and the other nodes set as servers.
     """
-    layout = {'controller' : "node-1"}
-    servers = []
-    num_slaves = self.min_vms - 1
-    for i in xrange(num_slaves):
-      servers.append("node-{0}".format(i+2))
-
-    layout['servers'] = servers
-    return layout
-
+    return {
+      'controller': "node-1",
+      'servers': ["node-{}".format(i)\
+                  for i in xrange(2, self.min_vms - 1)]
```

Comment:
If `min_vms` is 2, the controller would get one, but there would be nothing in `servers`.